plugins/gatsby-source-cd-airtable/src/create-nodes.ts

```diff
@@ -0,0 +1,24 @@
+import { NodeInput, SourceNodesArgs } from 'gatsby'
+import { SourceNode } from './interfaces/source-node'
+
+export const createNodesFactory = ({
```

Comment:
Přijde mi, že factory pattern je trochu složitější pro další úpravy, ale asi to není zásadní bod.